modules/ROOT/pages/tutorial/tutorial-sso-configuration.adoc

```diff
@@ -192,90 +192,90 @@ See xref:authentication-authorization/sso-integration.adoc#auth-sso-auth-provide
 
 == Microsoft Entra ID (formerly Azure Active Directory)
 
+The following examples show how to configure Microsoft Entra ID for authentication and authorization using access tokens and ID tokens.
+
+=== Register the application
+
+. Log in to the https://portal.azure.com/#home[Azure portal].
+. Click *Microsoft Entra ID* and navigate to *Manage -> App registrations*.
+. Click *New registration*.
+. Type a name for your application, for example, `Neo4j SSO`.
+. Under *Select the supported account types*, select `Accounts in this organizational directory only (Default Directory only - Single tenant)`.
+. Under *Redirect URI*, select `Single-page application (SPA)` and enter the redirect URI: `http://localhost:7474/browser/?idp_id=azure&auth_flow_step=redirect_uri`
+The redirect URI will accept the returned token responses after successful authentication.
+. Click *Register*.
+
 === Access token
 
-This example shows how to configure Entra ID for authentication and authorization using an access token.
+This example shows how to configure Neo4j to use an Entra ID access token for authentication and authorization.
+
+==== Configure Entra ID
 
-. After the successful creation of your SSO application in Azure, open the *Token configuration* tab to configure a token.
+. From the *App registrations* page, select the app you just created.
+. From the left-hand side menu, navigate to *Manage -> Token configuration*.
 .. Click *Add groups claim*.
-.. Select *Security groups* to include in your access token.
+.. Select *Groups assigned to the application (recommended for large enterprise companies to avoid exceeding the limit on the number of groups a token can emit)* to include in your access token.
 .. Save your changes.
 
-. Open the **Expose an API** tab and select **Add a Scope**.
-.. If you click the **Add a Scope** button for the first time, you see a new pane stating that you need to add an _Application ID URI_ before proceeding.
+. Navigate to *Expose an API* and click **Add a Scope**.
 +
+[NOTE]
+====
+The first time you click the *Add a Scope* button, you see a new pane stating that you need to add an _Application ID URI_ before proceeding.
 You can find it on your app *Overview* page.
-+
-.The GUID is used to identify specific resources or instances within Azure. You can find it on the app registration page.
-image::sso-configuration-tutorials/azure-id.svg[]
-+
-.. Click *Save and continue* after setting the _Application ID URI_.
+It is a GUID that looks like this: `api://<GUID>`.
+The GUID is a unique identifier for your application.
```

Comment:
If use of the `api://` prefix causes questions, we can add a sentence here clarifying that:  

The AUD parameter for access tokens is typically set with `api://` at the front. You can determine the use of `api://` via the tab **Edit AUD (Access token)** under the **Token configuration** entry.

@renetapopova, @mnd999, what do you think?